src/archive/2023/2023-04-20.md

```diff
@@ -8,6 +8,10 @@
 
     :material-map-marker: **Location:** St. Paul's Hospital ([1081 Burrard Street, Vancouver, BC V6Z 1Y6](https://goo.gl/maps/uCFbWCXcVrLmMnch7)), Cullen Family Lecture Theatre ([Providence Building, Level 1, Room 1477](https://docs.google.com/document/d/1xHHd14LcrDIZLG7RGBGneLgf12H-FJwpyH7rotQCo9k/edit?usp=sharing))
 
+    **RSVP**
+
+    If you are interested in attending this seminar, please fill out [the RSVP form](https://forms.gle/FZTes9e5gJcapAUC7).
```

Comment:
The form on the other end of this URL specifies "VanBUG seminar - Thursday, January 19th @ 6:00 PM" when it should be April info. It's a different link from the January file so I assume it just needs to be updated on the other end @kmnip ?

Reply:
Thanks for catching this! It is fixed now!